src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -251,6 +252,18 @@ def load_command_table(self, _):
         g.generic_update_command('update',
                                  custom_func_name='db_audit_policy_update')
 
+    server_blob_auditing_policies_operations = CliCommandType(
+        operations_tmpl='azure.mgmt.sql.operations#ServerBlobAuditingPoliciesOperations.{}',
+        client_factory=get_sql_server_blob_auditing_policies_operations)
+
+    with self.command_group('sql server audit-policy',
```

Comment:
could we mark these new command group as Preview for one sprint with `is_preview=True`?

Reply:
No problem, I'll add it on next commit.